src/posit/connect/content.py

```diff
@@ -229,6 +229,7 @@ def restart(self) -> None:
     @overload
     def update(
         self,
+        *args,
```

Comment:
are these *args additions just for forwards / backwards compatibility?

Reply:
These are required to match the dict.update signature. Since Resource inherits from dict, we have to adhere to it. You can see this by removing *args and running `make lint`.

Reply:
See below first. This is the equivalent to *, except in this case we have to include *args to match the update signature.